packages/example/src/pages/NonVirtualizedGridPage.tsx

```diff
@@ -0,0 +1,66 @@
+import { DefaultFocus, SpatialNavigationNode } from 'react-tv-space-navigation';
+import { Page } from '../components/Page';
+import '../components/configureRemoteControl';
+import { getPrograms } from '../modules/program/infra/programInfos';
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { RootStackParamList } from '../../App';
+import styled from '@emotion/native';
+import { Fragment } from 'react';
+import { scaledPixels } from '../design-system/helpers/scaledPixels';
+import { ProgramNode } from '../modules/program/view/ProgramNode';
+
+export const NonVirtualizedGridPage = () => {
+  return (
+    <Page>
+      <Container>
+        <SpatialNavigationNode alignInGrid>
+          <Fragment>
+            <DefaultFocus>
```

Comment:
You should put DefaultFocus around the whole grid. It will take the first element anyway.

Reply:
That's why the fragment, the node is designed to accept only one child. With default focus as I wrote it it didn't work so I had to put a fragment.
I'll do as you said in the other comment ;)